_dev/cldr/number-formatter.js

```diff
@@ -251,15 +252,20 @@ class CurrencyFormatter {
    * @return mixed
    */
   performSpecificReplacements(formattedNumber) {
-    return formattedNumber
-      .split(CURRENCY_SYMBOL_PLACEHOLDER)
-      .join(this.numberSpecification.getCurrencySymbol());
+    if (this.numberSpecification instanceof PriceSpecification) {
+      return formattedNumber
+        .split(CURRENCY_SYMBOL_PLACEHOLDER)
+        .join(this.numberSpecification.getCurrencySymbol());
+    }
+
+    return formattedNumber;
   }
 
   static build(specifications) {
     const symbol = new NumberSymbol(...specifications.symbol);
-    const currency = new CurrencyFormatter(
-      new PriceSpecification(
+    let specification;
+    if (specifications.currencySymbol) {
```

Comment:
You don't need to check if it exists? Like with `specifications.currencySymbol != undefined` or `typeof specifications.currencySymbol != undefined` ?

Reply:
I want to check for a value != null && != undefined

Reply:
Yes what I meant is if the `currencySymbol` property is not even present in `specifications`
Does this test work?

Reply:
Ok I tested it it works in all use cases, fine for me!